template/build/utils.js

```diff
@@ -20,9 +20,13 @@ exports.cssLoaders = function (options) {
     }
   }
 
+  var postcssLoader = {
+    loader: 'postcss-loader'
+  }
+
   // generate loader string to be used with extract text plugin
   function generateLoaders (loader, loaderOptions) {
-    var loaders = [cssLoader]
+    var loaders = options.usePostCSS !== false ? [cssLoader, postcssLoader] : [cssLoader]
```

Comment:
1. Where does `option.usePostCSS` come from? It's not to be found in `/config/index.js`, not in any of the webpack config files. This should be documented someway.

Is the idea to simply add it to the styleLoader code like this?

```javascript
// in webpack.base.conf.js:

rules: utils.styleLoaders({ sourceMap: config.dev.cssSourceMap, usePostCSS: true })
```

Reply:
> Is the idea to simply add it to the styleLoader code like this?
>
> ```js
> // in webpack.base.conf.js:
>
> rules: utils.styleLoaders({ sourceMap: config.dev.cssSourceMap, usePostCSS: true })
> ```

Yes, I think the `usePostCSS` config should be set in the webpack config file.